site/hxapi/transfers/av/index.md

```diff
@@ -198,7 +200,8 @@ Here are the available categories which may be returned, there will only be one
                     "JoiningInstructions": null,
                     "ContactNumbers": null
                 },
-                "advance_purchase": true
+                "advance_purchase": true,
+                "partnerOwnSupply": false
```

Comment:
When I test https://github.com/holidayextras/hapi/pull/8887 with `.js` suffix, I see `"PartnerOwnSupply":""`
I think the Pascal case version is more consistent with the rest of the fields, so that probably needs updating here. (Though the empty string is surely incorrect, but that's one for 8887.)

I also see it above `advance_purchase` (admittedly less important, but may as well make it match the actual response order, to make it easy for users).